sdks/android/Makefile

```diff
@@ -160,16 +188,64 @@ package: $(patsubst %,$(APK_DIR)/jni/arm64-v8a/%,$(MONO_LIBS))
 package: $(patsubst %,$(APK_DIR)/jni/x86/%,$(MONO_LIBS))
 package: $(patsubst %,$(APK_DIR)/jni/x86_64/%,$(MONO_LIBS))
 package: $(APK_DIR)/assets/mconfig
-package: $(APK_DIR)/assets/asm/main.exe
 package: $(APK_DIR)/assets/asm/Mono.Android.dll
 package: $(patsubst %,$(APK_DIR)/assets/asm/%,$(BCL_LIBS))
+package: $(patsubst %,$(APK_DIR)/assets/asm/%,$(BCL_PDBS))
 package: $(patsubst %,$(APK_DIR)/assets/asm/%,$(NUNIT_LIBS))
 package: $(patsubst %,$(APK_DIR)/assets/asm/monodroid_%_test.dll,$(CHECKS))
 
+package-bcl: package
+	rm -rf $(APK_DIR)/bin/AndroidActivity-debug.apk
+	rm -rf $(DEBUGGER_RUNNER_EXE)
+	make $(BCL_RUNNER_EXE)
+
+package-debugger: package
+	rm -rf $(BCL_RUNNER_EXE)
+	rm -rf $(APK_DIR)/bin/AndroidActivity-debug.apk
+	rm -rf $(APK_DIR)/assets/asm/testassembly.txt
+	make $(DEBUGGER_RUNNER_EXE)
+	make $(DEBUGGER_RUNNER_PDB)
+
+.PHONY: $(TOP)/mcs/class/Mono.Debugger.Soft/dtest-app.exe $(TOP)/mcs/class/Mono.Debugger.Soft/dtest-app.pdb
```

Comment:
why are those `.PHONY` targets?

Reply:
You don't know when they should be rebuilt, since the knowledge resides in the `mcs/` build system so to avoid missing a necessary rebuild, you always rebuild (i.e. it's a "phony" target)